misc/custom-flows-environment-variables/env0.yml

```diff
@@ -5,7 +5,13 @@ deploy:
     terraformPlan:
       before:
         - echo USER_NAME=user >> $ENV0_ENV
+        - echo FULL_NAME_WITH_WHITESPACE=\"John Doe\" >> $ENV0_ENV
         - ./export-user-id.sh
       after:
         - echo $USER_NAME
+        - echo $FULL_NAME_WITH_WHITESPACE
         - ./print-user-id.sh
+        - echo "Asserting expected values..."
+        - "[ $USER_NAME == user ]"
```

Comment:
🟡 
if it fails it fails the deploy, right?

Reply:
yes